server/manager/src/main/java/org/apache/accumulo/master/recovery/RecoveryManager.java

```diff
@@ -171,6 +171,8 @@ public boolean recoverLogs(KeyExtent extent, Collection<Collection<String>> walo
         String dest =
             RecoveryPath.getRecoveryPath(new Path(filename), master.getContext()).toString();
 
+        log.debug("This is walPath in RecoveryManager: " + filename);
+
```

Comment:
You could use parameter replacement in log statements
`log.debug("This is walPath in RecoveryManager: {}", filename);`

Reply:
Thank you EdColeman, I am making the change now.

Reply:
I see you updated this log statement in the latest commit. Looks like there may be a couple others spots that could be changed to use this as well for the sake of consistency.